src/test_workflow/integ_test/local_test_cluster.py

```diff
@@ -4,19 +4,11 @@
 # this file be licensed under the Apache-2.0 license or a
 # compatible open source license.
 
-import logging
 import os
-import subprocess
-import time
 
-import requests
-import yaml
-
-from paths.tree_walker import walk
-from system.process import Process
-from test_workflow.test_cluster import ClusterCreationException, TestCluster
+from test_workflow.integ_test.service_opensearch import ServiceOpenSearch
+from test_workflow.test_cluster import TestCluster
 from test_workflow.test_recorder.test_recorder import TestRecorder
-from test_workflow.test_recorder.test_result_data import TestResultData
 
 
 class LocalTestCluster(TestCluster):
```

Comment:
Callers of LocalTestCluster should be passing in a way to control which service should be started, are those interface changes coming separately?

Reply:
#838 under the main issue #604 will cover this part.